ports/arrow/vcpkg.json

```
@@ -0,0 +1,87 @@
+{
+  "name": "arrow",
+  "version": "4.0.0",
+  "description": "Cross-language development platform for in-memory analytics",
+  "homepage": "https://arrow.apache.org",
```

```suggestion
  "homepage": "https://arrow.apache.org",
  "supports": "x64",
```
The previous supports clause should be preserved